test/NIST-800-53/NIST-800-53-CloudTrail.test.ts

```diff
@@ -0,0 +1,241 @@
+          }),
+        })
+      );
+    });
+
+    test('NIST.800.53-CloudTrailLogFileValidationEnabled: Cloud Trails have log file validation enabled', () => {
+      //AC 1: Given a CDK stack with one or more non-compliant Cloud Trails
+      //when NIST-503 Secure Aspects is run
+      //the CDK stack does not deploy and the consultant receives an explanation about the non compliant trail for the relevant NIST standards
+      const nonCompliant = new Stack();
+      Aspects.of(nonCompliant).add(new NIST80053Checks());
+
+      const myLogs = new LogGroup(nonCompliant, 'rLogGroup');
+
+      const trail = new Trail(nonCompliant, 'rTrail', {
+        cloudWatchLogGroup: myLogs,
+        sendToCloudWatchLogs: true,
+        enableFileValidation: false,
+      });
+
+      trail.stack;
+
+      const messages1 = SynthUtils.synthesize(nonCompliant).messages;
+      expect(messages1).toContainEqual(
+        expect.objectContaining({
+          entry: expect.objectContaining({
+            data: expect.stringContaining(
+              'NIST.800.53-CloudTrailLogFileValidationEnabled:'
+            ),
+          }),
+        })
+      );
+
+      //AC 2:
+      //Given a CDK stack with compliant Cloud Trails(s):
+      //When NIST-503 Secure Aspects is run
+      //Then the CDK stack deploys and the consultant does not receive an explanation about the compliant resource for the relevant NIST standards
+      const activeCompliant = new Stack();
+      Aspects.of(activeCompliant).add(new NIST80053Checks());
+
+      const myLogs2 = new LogGroup(activeCompliant, 'rLogGroup');
+
+      const myKey = new Key(activeCompliant, 'rKey');
+
+      const trail2 = new Trail(activeCompliant, 'rTrail', {
+        cloudWatchLogGroup: myLogs2,
+        sendToCloudWatchLogs: true,
+        kmsKey: myKey,
+      });
+
+      trail2.logAllLambdaDataEvents();
+
+      const messages2 = SynthUtils.synthesize(activeCompliant).messages;
+      expect(messages2).not.toContainEqual(
+        expect.objectContaining({
+          entry: expect.objectContaining({
+            data: expect.stringContaining(
+              'NIST.800.53-CloudTrailLogFileValidationEnabled:'
+            ),
+          }),
+        })
+      );
+
+      //AC 3:
+      //Given a CDK stack with no Cloud Trails:
+      //When NIST-503 Secure Aspects is run
+      //Then the CDK stack deploys and the consultant does not receive an explanation about the revelant NIST standards
+      const passiveCompliant = new Stack();
+      Aspects.of(passiveCompliant).add(new NIST80053Checks());
+
+      //no Cloud Trail resources
+
+      const messages3 = SynthUtils.synthesize(passiveCompliant).messages;
+      expect(messages3).not.toContainEqual(
+        expect.objectContaining({
+          entry: expect.objectContaining({
+            data: expect.stringContaining(
+              'NIST.800.53-CloudTrailLogFileValidationEnabled:'
+            ),
+          }),
+        })
+      );
+    });
+  });
+});
```

Comment:
The engine tests check for the passive compliance. I don't think we need to check for passive compliance
```suggestion
```